### PR TITLE
CS-1048 HelperModule

### PR DESCRIPTION
**Pull request by EricStMartin**
Helper Modules part 1: Pre-Requisites for new DVLS Server including offline mode.  
Public commands: 
Start-DVLSSetup
New-OfflineDeployment

### CODE REVIEW

File: HelperModule/Public/New-OfflineDeployment.ps1 [hunk 1]
```diff
@@ -0,0 +1,30 @@
+﻿function New-OfflineDeployment {
+    New-EventSource
```
Comment (EricStMartin) [inline HelperModule/Public/New-OfflineDeployment.ps1:2]: Do we maybe remove this as we do not really want to write to the EventLog on their PCs... ?

File: HelperModule/DVLS.HelperModule.psd1 [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    # Format files (.ps1xml) to be loaded when importing this module
+    # FormatsToProcess = @()
+
+    # Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
+    NestedModules        = @()
+
+    # Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
+    FunctionsToExport    = @('Get-DevoConsoleVersion', 'Get-SQLDBVersion', 'Install-DevolutionsServer', 'Install-DVLSConsole', 
+        'Install-DVLSInstance', 'Install-SQLServer', 'Install-SSMS', 'New-OfflineDeployment', 'New-OfflineServer', 'Start-PrerequisiteSetup', 
+        'Update-DVLSInstace', 'Update-SQLDatabase')
```
Comment (mauricecote) [inline HelperModule/DVLS.HelperModule.psd1:74]: typo in Instance

Comment (EricStMartin) [reply HelperModule/DVLS.HelperModule.psd1]: Fixed

File: HelperModule/DVLS.HelperModule.psd1 [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+    # Format files (.ps1xml) to be loaded when importing this module
+    # FormatsToProcess = @()
+
+    # Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
+    NestedModules        = @()
+
+    # Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
+    FunctionsToExport    = @('Get-DevoConsoleVersion', 'Get-SQLDBVersion', 'Install-DevolutionsServer', 'Install-DVLSConsole', 
+        'Install-DVLSInstance', 'Install-SQLServer', 'Install-SSMS', 'New-OfflineDeployment', 'New-OfflineServer', 'Start-PrerequisiteSetup',
```
Comment (mauricecote) [inline HelperModule/DVLS.HelperModule.psd1:73]: le start-prerequisiteSetup devrait être le verb Install, non?

File: HelperModule/DVLS.HelperModule.psm1 [hunk 4]
```diff
@@ -0,0 +1,27 @@
+#Requires -RunAsAdministrator
+using namespace System
+using namespace System.Diagnostics
+
+Set-StrictMode
+#Set-StrictMode -Version 1.0
```
Comment (mauricecote) [inline HelperModule/DVLS.HelperModule.psm1:6]: delete

File: HelperModule/DVLS.HelperModule.psd1 (hunk 3 above)
Comment (EricStMartin) [reply HelperModule/DVLS.HelperModule.psd1:73]: J'imagine que oui pour suvire les autres

File: HelperModule/Public/New-OfflineDeployment.ps1 (hunk 1 above)
Comment (mauricecote) [reply HelperModule/Public/New-OfflineDeployment.ps1:2]: ah, good catch, its not needed on a "client" pc

**Review by mauricecote — APPROVED** [review]
minor issues noted in single line comments, but its a good start

**Pull request merged**